_templates/feature/new/default-entry.ejs.t

```diff
@@ -0,0 +1,42 @@
+---
+# helper funcs https://www.hygen.io/docs/templates/#helpers-and-inflections
+to: blocks/<%= h.inflection.dasherize(block_name) %>-block/features/<%= h.inflection.dasherize(block_name) %>/default.jsx
+---
+import React from 'react';
+import PropTypes from '@arc-fusion/prop-types';
+
+// icons and other utilities can be found in the engine theme sdk storybook
+// https://beta--5eed0506faad4f0022fedf95.chromatic.com/
+import {
+  EnvelopeIcon,
+} from '@wpmedia/engine-theme-sdk';
+
+function <%= h.changeCase.pascal(block_name) %>(props) {
```

Comment:
Can we switch the syntax to be

```
const <%= h.changeCase.pascal(block_name) %> = ({ customFields }) => {
```

this makes it more consistent with how existing blocks are built

Reply:
since client devs may not be familiar with props, didn't want to do too much destructuring. I also put in an explainer with what props are

Reply:
This is within our repo - and should act as a "best practise"

Reply:
updated